flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/backends/hdfs/HDFSBackendImpl.java

```diff
@@ -237,4 +258,95 @@ private HttpResponse doHDFSRequest(String method, String url, ArrayList<Header>
         return response;
     } // doHDFSRequest
     
+    // from here on, consider this link:
+    // http://stackoverflow.com/questions/21629132/httpclient-set-credentials-for-kerberos-authentication
+    private HttpResponse doPrivilegedHDFSRequest(String method, String url, ArrayList<Header> headers,
+            StringEntity entity) throws Exception {
+        try {
+            LoginContext loginContext = new LoginContext("cygnus_krb5_login",
```

Comment:
Yes, it is hardwired since the `krb5_login.conf` file must not be edited (as stated in the documentation).